Mathlib/Topology/LocallyFinsupp.lean

```diff
@@ -322,13 +327,49 @@ instance [Lattice Y] [Zero Y] : Lattice (locallyFinsuppWithin U Y) where
   inf_le_right D₁ D₂ := fun x ↦ by simp
   le_inf D₁ D₂ D₃ h₁₃ h₂₃ := fun x ↦ by simp [h₁₃ x, h₂₃ x]
 
+variable [AddCommGroup Y]
+
+@[simp] lemma posPart_apply (a : locallyFinsuppWithin U Y) (x : X) : a⁺ x = (a x)⁺ := rfl
+@[simp] lemma negPart_apply (a : locallyFinsuppWithin U Y) (x : X) : a⁻ x = (a x)⁻ := rfl
+
+end Lattice
+
+section LinearOrder
+variable [AddCommGroup Y] [LinearOrder Y] [IsOrderedAddMonoid Y]
+
 /--
 Functions with locally finite support within `U` form an ordered commutative group.
 -/
-instance [AddCommGroup Y] [LinearOrder Y] [IsOrderedAddMonoid Y] :
-    IsOrderedAddMonoid (locallyFinsuppWithin U Y) where
+instance : IsOrderedAddMonoid (locallyFinsuppWithin U Y) where
   add_le_add_left := fun _ _ _ _ ↦ by simpa [le_def]
 
+/--
+Taking the positive part of a function with locally finite support commutes with
+scalar multiplication by a natural number.
+-/
+@[simp]
+theorem nsmul_posPart (f : locallyFinsuppWithin U Y) (n : ℕ) : (n • f)⁺ = n • f⁺ := by
```

Comment:
```suggestion
theorem nsmul_posPart (n : ℕ) (f : locallyFinsuppWithin U Y) : (n • f)⁺ = n • f⁺ := by
```
to respect the order in which things appear in the lemma

Reply:
Done. Thanks for the reminder.